builder/builder.go

```diff
@@ -128,6 +128,14 @@ type Docker interface {
 	// Release releases a list of images that were retained for the time of a build.
 	// TODO: remove
 	Release(sessionID string, activeImages []string)
+	// Kill stops the container execution abruptly.
+	Kill(c *daemon.Container) error
+	// Mount mounts the root filesystem for the container.
+	Mount(c *daemon.Container) error
+	// Unmount unmounts the root filesystem for the container.
+	Unmount(c *daemon.Container) error
+	// Start starts a new container
+	Start(c *daemon.Container) error
```

Comment:
we're adding these function to the builder because the builder relies on calling the daemon internally for those operations. The builder builds images based on containers. We don't want to do a lookup internally because we already have the container booted when these functions are called.

After these changes are merged we'll remove the container struct from the daemon package.